renku_notebooks/util/kubernetes_.py

```diff
@@ -73,6 +75,17 @@ def make_server_name(
     )
 
 
+def renku_1_make_server_name(safe_username: str, project_id: str, environment_id: str) -> str:
+    """Form a unique server name."""
+    username_hash = md5(safe_username.encode()).hexdigest().lower()
+    environment_id_hash = md5(environment_id.encode()).hexdigest().lower()
+    project_id = project_id.lower()
+
+    # NOTE: A K8s object name can only contain lowercase alphanumeric characters, hyphens, or dots.
+    # Must be less than 253 characters long and start and end with an alphanumeric.
+    return f"renku-1-{username_hash}-{project_id}-{environment_id_hash}"
```

Comment:
There should probably a randomized small string at the end to support multiple sessions on the same launcher.